src/instructlab/cli/data/data.py

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.configuration import storage_dirs_exist
+
+
+@click.group(
+    cls=clickext.LazyEntryPointGroup,
+    ep_group="instructlab.command.data",
+)
+@click.pass_context
+def data(ctx):
+    """Command Group for Interacting with the Data generated by InstructLab.
+
+    If this is your first time running ilab, it's best to start with `ilab config init` to create the environment.
+    """
+    ctx.obj = ctx.parent.obj
```

Comment:
can we do the ctx assigning after the storage check?

Reply:
I haven't touched the code in this file, only moved it over

Reply:
If that's the case let's do it as a followup - I'd like to change as little as possible following the large-size E2E success